content_script.js

```diff
@@ -176,6 +166,31 @@ class HTMLSerializer {
       return fullyQualifiedName + '.' + index; 
     }
   }
+
+  /**
+   * Calculate the correct quotes that should be used given how many parent
```

Comment:
Instead of "how many parent windows", I'd say "the number of nested frames between this one and the root frame".

Also, replace "windows" with "frames" as that is the usual terminology.